modules/iam/outputs.tf

```diff
@@ -1,8 +1,10 @@
 output "config" {
   value = {
-    service_role = aws_iam_role.eks_service_role.name
-    node_role    = aws_iam_role.eks_node.name
+    service_role_arn = aws_iam_role.eks_service_role.arn
+    node_role        = aws_iam_role.eks_node.name
```

Comment:
Can we loose this?

Reply:
this role name is still used by [the instance profile](https://github.com/cookpad/terraform-aws-eks/blob/118ad3076872b052e6cc293ea3be500c840a9b21/modules/cluster/outputs.tf#L9-L10). Otherwise, we still need to retrieve the name using `data` sources. I will consider if I can drop the variable with another way.

Reply:
removed with https://github.com/cookpad/terraform-aws-eks/pull/202/commits/33bef42b4a08e247a17ffde92337c4411835d457